src/tests/performVerifySorting.spec.js

```diff
@@ -0,0 +1,38 @@
+const { expect } = require('@playwright/test');
+const { test } = require('../fixture');
+const { login, password } = require('../auth/user.json');
+
+test.describe('Inventory Sorting', () => {
+    const testParams = [
+        { name: 'az', sort: [] },
+        { name: 'za', sort: [] },
+        { name: 'lohi', sort: [] },
+        { name: 'hilo', sort: [] },
+    ];
+
+    test.beforeEach(async ({ inventoryPage, loginPage }) => {
+        await inventoryPage.navigate();
+        await loginPage.performLogin(login, password);
+
+        const itemNames = await inventoryPage.getAllProductNames();
+        testParams[0].sort = [...itemNames].sort();
+        testParams[1].sort = [...itemNames].sort().reverse();
+
+        const itemPrices = await inventoryPage.getAllPrices();
+        testParams[2].sort = [...itemPrices].sort((a, b) => a - b);
+        testParams[3].sort = [...itemPrices].sort((a, b) => b - a);
```

Comment:
it will be done very time before each test. Do we need to do it 4 times, I mean the same